_delphi_utils_python/tests/test_covidcast_wrapper.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta
+from pathlib import Path
+import pandas as pd
+from delphi_utils import covidcast_wrapper
+import covidcast
+from freezegun import freeze_time
+from pandas.testing import assert_frame_equal
+
+TEST_DIR = Path(__file__).parent
+class TestCovidcastWrapper:
+    def test_metadata(self):
+        expected_df = covidcast.metadata()
+        df = covidcast_wrapper.metadata()
+        assert_frame_equal(expected_df, df)
+
+    @freeze_time("2024-07-29")
+    def test_signal(self):
+        meta_df = covidcast_wrapper.metadata()
+        data_filter = ((meta_df["max_time"] >= datetime(year=2024, month=6, day=1)) & (meta_df["time_type"] == "day"))
```

Comment:
The way you are filtering this looks like it will include new signals that are added...  `@freeze_time` only applies to the local scope of this test, not what youre gonna get back from the live API!